generated/cached-endpoints.edn

```diff
@@ -1552,7 +1521,13 @@
     :responses
     [{:status 200,
       :description "Grant a user access to a product",
-      :type "user-product-access"}]},
+      :type "user-product-access"}
+     {:status 400,
```

Comment:
Isn't 400 for BadRequest instead of granting access? My hunch says it d be 401 in that case. But might be wrong